# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/sysctl-x32.patch
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We do not want to e.g. libiberty to fail, especially as it already
detects to nicely that sysctl is not present anyway, ...

	- Rene Rebe

--- glibc-2.22/sysdeps/unix/sysv/linux/x86/bits/sysctl.h.orig	2016-01-07 20:31:08.011863995 +0000
+++ glibc-2.22/sysdeps/unix/sysv/linux/x86/bits/sysctl.h	2016-01-07 20:31:17.331864170 +0000
@@ -16,5 +16,5 @@
    <http://www.gnu.org/licenses/>.  */
 
 #if defined __x86_64__ && defined __ILP32__
-# error "sysctl system call is unsupported in x32 kernel"
+# warning "sysctl system call is unsupported in x32 kernel"
 #endif
